torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1554,6 +1554,42 @@ int listRemove(Stack& stack) {
   return 0;
 }
 
+template <typename T>
+int listMin(Stack& stack) {
```

Comment:
The copy paste isn't great, it could be 1 method that returns a `std::tuple(min, max)` then when used you can statically `.get<1>` or `.get<0>`

Reply:
See discussion here: https://github.com/pytorch/pytorch/pull/26155#discussion_r324294589

Reply:
Also i don't think doing twice as many comparisons is worth it either

Reply:
We might not necessarily save a whole lot here, since we would probably need lambda wrappers and `get<0>`, `get<1>` aren't very descriptive even in the context of `Operator("min`.  We are probably very unlikely to ever hit an issue with these two functions to warrant the deduplication of the two?